UPGRADE-7.1.md

```diff
@@ -0,0 +1,7 @@
+UPGRADE FROM 7.0 to 7.1
+=======================
+
+HttpKernel
+----------
+
+ * `ExceptionEvent` now takes an optional `$isKernelTerminating` parameter
```

Comment:
Shouldn't we add this to the CHANGELOG instead to the upgrade guide, as this is an optional one?